std/dynamic_library.zig

```diff
@@ -261,43 +282,57 @@ pub const WindowsDynLib = struct {
         self.* = undefined;
     }
 
-    pub fn lookup(self: *WindowsDynLib, name: []const u8) ?usize {
-        if (windows.kernel32.GetProcAddress(self.dll, name.ptr)) |addr| {
-            return @ptrToInt(addr);
+    pub fn lookup(self: *WindowsDynLib, comptime T: type, name: []const u8) !?T {
+        const c_name: [512]u8 = []u8{0} ** 512;
```

Comment:
It's not a big deal but this requires setting all the bytes to zeroes. It's not free since this isn't a global variable. If you leave it undefined and only set 1 zero at the end, then less memory has to be accessed for smaller names.